src/olympia/reviewers/forms.py

```diff
@@ -285,8 +331,8 @@ class ReviewForm(forms.Form):
                                label=_(u'Comments:'))
     canned_response = NonValidatingChoiceField(required=False)
     action = forms.ChoiceField(required=True, widget=forms.RadioSelect())
-    versions = ModelMultipleChoiceField(
-        widget=forms.SelectMultiple(attrs={'class': 'data-toggle'}),
+    versions = VersionsChoiceField(
+        widget=VersionsChoiceWidget(attrs={'class': 'data-toggle'}),
```

Comment:
so we're still setting the class to `data-toggle` here _and_ in `VersionsChoiceWidget.create_option`?  Are they setting different tags?

Reply:
Yeah. This one is affecting the `<select>` - which shouldn't be shown for all actions. The ones I've added are for the individual `<option>` inside the `<select>`, since for unlisted there are some actions where we don't want to see all versions.

I'll add a comment.

Reply:
bc7f7d9cdbafcf95fce5f57310bc24393d71f654